admin_manual/enterprise_user_management/file_firewall.rst

```diff
@@ -0,0 +1,174 @@
+all conditions evaluate to true. If a request matches at least one of the 
+defined rules, the request is blocked.
+
+Administrators are excluded from the firewall when they 
+access the instance via the WebUI, to allow admins to modify the rules. 
+Otherwise they could potentially be locked out. Public share links and syncing 
+via WebDAV are subject to firewall rules for all users all the time.
+
+Figure 1 shows an empty firewall configuration panel. Set your logging level to 
+**Failures Only** for debugging, and create a new ruleset by clicking the **Add 
+Group** button. After setting up your rules you must click the **Save Rules** 
+button.
+
+.. figure:: images/firewall-1.png
+   :alt: Empty File Firewall configuration panel.
+   
+   *Figure 1: Empty File Firewall configuration panel*
+
+Figure 2 shows two rules. The first rule, **No Support outside 
+office hours**, prevents members of the support group from logging into the 
+ownCloud Web interface from 5pm-9am.
+
+The second rule prevents members of the qa-team group from accessing the Web UI 
+from IP addresses that are outside of the local network.
+
+.. figure:: images/firewall-2.png
+   :alt: Two example rules that restrict logins per user group.
+   
+   *Figure 2: Two example rules that restrict logins per user group*   
+
+All other users are not affected, and can log in anytime from anywhere.
+
+Available Conditions
+--------------------
+
+User Group
+ The user (is|is not) a member of the selected group.
+
+User Agent
+ The User-Agent of the request (matches|does not match) the given string.
+
+User Device
+  A shortcut for matching all known (`android`|`ios`|`desktop`) sync clients by 
+  their User Agent string.
+
+Request Time
+ The time of the request (has to|must not) in a single range from beginning 
+ time to end time.
+
+Request URL
+ The **full page URL** (has to|must not) (match|contain|begin with|end) with a 
+ given string.
+
+Request Type
+ The request (is|is not) a (WebDAV|public share link|other) request.
+
+Request IP Range
+ The request's ``REMOTE_ADDR`` header (is|is not) matching the given IP range.
+
+Subnet
+ The request's ``SERVER_ADDR`` header (is|is not) matching the given IP range.
+
+File Size Upload
+ When a file is uploaded the size has to be (less|less or equal|greater|greater 
+ or equal) to the given size.
+
+File Mimetype Upload
+ When a file is uploaded the mimetype (is|is not|begins with|does not begin 
+ with|ends with|does not end with) the given string.
+
+Regular Expression
+ The File Firewall supports regular expressions, allowing you to create custom 
+ rules using the following conditions:
+
+* User IP
+* Subnet
+* User agent
+* User group
+* Request URL
+
+You can combine multiple rules into one rule. E.g., if a rule applies to both 
+the support and the qa-team you could write your rule like this::
+
+ Regular Expression > `^(support|qa-team)$` > `is` > `User group`
```

Comment:
Are they unnecessary? I want to show the literal example exactly as it should appear.